elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/api/registry/JobInstanceRegistry.java

```diff
@@ -17,9 +17,14 @@
 
 package org.apache.shardingsphere.elasticjob.lite.api.registry;
 
+import java.util.Arrays;
```

Comment:
There are too many unrelated changes in this PR. Please revert them.

Reply:
Caused by reformatting code.